### PR TITLE
- ST7735 TFT display support - port from SmingNONOS

### PR DESCRIPTION
**Pull request by harry-boe**
- undefine debug flag for HW SPI impl

Tested on SW and HW SPI (std setting 4Mhz bus speed)

### CODE REVIEW

**Comment by harry-boe** [comment]
well that's weird - the CI build is missing a include file that is part of the PR

no idea how to fix this ?!

File: samples/ScreenTFT_ST7735/app/BMPDraw.cpp [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+/****************************************************
+ * ported for Sming by H.Boettcher.
+ * this implementation uses spifs storage instead of SD cards
+ * hbottc@gmail.com
+ ***************************************************/
+
+
+#include <user_config.h>
+//#include <SmingCore/SmingCore.h>
+#include <Libraries/Adafruit_ST7735/Adafruit_ST7735.h>
```
Comment (alonewolfx2) [inline samples/ScreenTFT_ST7735/app/BMPDraw.cpp]: Library folder name must be  lowercase İ think

File: samples/ScreenTFT_ST7735/app/application.cpp [hunk 2]
```diff
@@ -0,0 +1,340 @@
+#include <user_config.h>
+//#include <SmingCore/SmingCore.h>
+#include <Libraries/Adafruit_ST7735/Adafruit_ST7735.h>
```
Comment (alonewolfx2) [inline samples/ScreenTFT_ST7735/app/application.cpp]: One more lowercase

**Comment by harry-boe** [comment]
Ok this happens when you use a non Case sensitive filesystem - THX for the hint

**Pull request merged**